lib/absinthe/phase/document/complexity.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Absinthe.Phase.Document.Complexity do
+  @moduledoc false
+
+  # Analyses document complexity.
+
+  alias Absinthe.{Blueprint, Phase}
+
+  use Absinthe.Phase
+
+  @default_complexity 1
```

Comment:
The default resolution function is essentially just a Map.get, and it is for all intents and purposes impossible to resolve any faster than that. Thus I think a default complexity of 1 isn't so bad, since it doesn't seem feasible to go lower anyway.